### ISSUE TITLE
Is there a reason why update_master_grads()

### ISSUE THREAD

**Issue by vince62s** [NVIDIA/apex#141]
was not implemented in the apex.optimizers.FP16_Optimizer class ? (the cut-down one)

thanks.

**Comment by mcarilli** [comment]
`apex.optimizers.FP16_Optimizer` was written separately, for the narrow use case of interacting with FusedAdam.  I guess they didn't need gradient accumulation for the original cases.

Apex came together out of a set of tools that developed separately but were similar enough that it made sense to put them all in once place.  We've been pleasantly surprised by the community uptake (frankly, it's been more than I can handle, and we are hiring another full timer to manage community engagement).  Given that Apex seems to be becoming popular, my main task right now is working on creating a unified API for the mixed precision tools (Amp, fp16_utils.fp16_optimizer, and optimizers.fp16_optimizer).  This will include supporting gradient accumulation for the different optimizers.  https://github.com/NVIDIA/apex/commits/api_refactor is tracking the changes (NOT yet functional by any means).  I'm hoping to present the new API in [a webinar Feb 20](https://info.nvidia.com/webinar-mixed-precision-with-pytorch-reg-page.html?ncid=so-lin-dplgdrd3-74473), and again at GTC San Jose.
